.travis.yml

```diff
@@ -8,19 +8,27 @@ before_install:
 
 install: true
 
+cache:
+  directories:
+    - $HOME/Library/Caches/org.carthage.CarthageKit/dependencies
+    - Carthage/Build
+
 jobs:
   include:
     - stage: Build + Test
       env:
+        - XCODE_WORKSPACE=Alicerce
```

Comment:
huuummm not sure, but don't you have to use `./Alicerce.xcworkspace` ? 🤔

Reply:
I do. I gave up yesterday after so many little fixes to make Travis happy. Fixing this now.